x-pack/plugins/monitoring/public/components/shared/toolbar.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiSuperDatePicker, OnRefreshChangeProps } from '@elastic/eui';
+import React, { useContext, useCallback } from 'react';
+import { MonitoringTimeContainer } from '../../application/pages/use_monitoring_time';
+
+export const MonitoringToolbar = () => {
+  const {
+    currentTimerange,
+    handleTimeChange,
+    setRefreshInterval,
+    refreshInterval,
+    setIsPaused,
+    isPaused,
+  } = useContext(MonitoringTimeContainer.Context);
+
+  const onTimeChange = useCallback(
+    (selectedTime: { start: string; end: string; isInvalid: boolean }) => {
+      if (selectedTime.isInvalid) {
+        return;
+      }
+      handleTimeChange(selectedTime.start, selectedTime.end);
+    },
+    [handleTimeChange]
+  );
+
+  const onRefreshChange = useCallback(
+    ({ refreshInterval: ri, isPaused: isP }: OnRefreshChangeProps) => {
+      setRefreshInterval(ri);
+      setIsPaused(isP);
+    },
+    [setRefreshInterval, setIsPaused]
+  );
+
+  return (
+    <EuiFlexGroup gutterSize={'xl'} justifyContent={'spaceBetween'}>
+      <EuiFlexItem>Setup Button</EuiFlexItem>
```

Comment:
For later, but looks like the NoData screen doesn't show the setup button. Guessing we'll need a flag to hide it.

Reply:
@matschaffer the setup button should not be rendered from the template. It's rendered from the [setup lib](https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/public/lib/setup_mode.tsx#L175).
In the page template, we only need the container that is used from that method, so the only thing we need to do is some cleanup here.